web/sites/example.settings.local.php

```diff
@@ -131,15 +131,15 @@
 $settings['skip_permissions_hardening'] = TRUE;
 
 /**
- * Exclude modules from configuration synchronisation.
+ * Exclude modules from configuration synchronization.
```

Comment:
Why change this from English to American ?

Reply:
I wondered that myself!